components/body/desktop.styl

```diff
@@ -44,24 +38,19 @@
   .body-section-content
     border-top-style none
 
-  .columns
-  .section1-text
-  .section-three-text-container
-  .section-five-content__part2
+  #genome img
+  .left-gutter-formatter
     padding-left 35px
     width 50%
-  .section-two-text
-  .carousel-track__image
-  .section-three img
-  .section-five-content__part1
-  .flex-right-formatting
+
+  #collecting img
+  .right-gutter-formatter
```

Comment:
When I was just using the class `.right-gutter-formatter` , that particular element was't being selected :-/